www/styles/LogReg.module.css

```diff
@@ -0,0 +1,57 @@
+.formContainer {
+  display: flex;
+  flex-direction: column;
+  margin: auto;
+  position: relative;
```

Comment:
In the real app we should try not to use `position: relative` as it is very difficult to debug.